android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/RatingBar.kt

```diff
@@ -0,0 +1,113 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import vn.luongvo.kmm.survey.android.ui.preview.SurveyDetailParameterProvider
+import vn.luongvo.kmm.survey.android.ui.screens.survey.AnswerUiModel
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.dimensions
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.typography
+
+private const val EMOJI_STAR = "⭐️️"
+private const val EMOJI_HEART = "❤️"
+
+private const val EMOJI_ALPHA_SELECTED = 1f
+private const val EMOJI_ALPHA_UNSELECTED = 0.5f
+
+@Composable
+fun StarRatingBar(
+    modifier: Modifier = Modifier,
+    answers: List<AnswerUiModel>,
+    onValueChange: (String) -> Unit,
```

Comment:
modifier should not be the first parameter but the first `optional` parameter
```suggestion
    answers: List<AnswerUiModel>,
    onValueChange: (String) -> Unit,
    modifier: Modifier = Modifier,
```

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/76/commits/cc081c374b5f5d77ec7cd0450559c9b48466e038